src/modules/interactive.js

```diff
@@ -0,0 +1,24 @@
+import UpdateUi from './mainSec.js';
+
+// status update
+
+export function clear() {
+  const newArr = UpdateUi.taskArr.filter((tasker) => tasker.complete !== true);
+  localStorage.setItem('taskItems', JSON.stringify(newArr));
+  UpdateUi.addTasks(newArr);
+  location.reload();
+}
```

Comment:
- [ ] The design of the clear button does not match that of the design. I would suggest giving a width of 100% and height that corresponds to that of the container